src/tilemap/TileLayer.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Threading.Tasks.Dataflow;
+
+namespace Tile_Mapper.tilemap
```

Comment:
The namespace is TileMapper, not Tile_Mapper. You also do not need a subnamespace for tilemap since it's just part of the main TileMapper one like TileSet is.